translator/translate/otel/exporter/otel_aws_cloudwatch_logs/translator.go

```diff
@@ -94,6 +94,9 @@ func (t *translator) Translate(c *confmap.Conf) (component.Config, error) {
 	if credentialsFileKey, ok := agent.Global_Config.Credentials[agent.CredentialsFile_Key]; ok {
 		cfg.AWSSessionSettings.SharedCredentialsFile = []string{fmt.Sprintf("%v", credentialsFileKey)}
 	}
+	if context.CurrentContext().Mode() == config.ModeOnPrem || context.CurrentContext().Mode() == config.ModeOnPremise {
```

Comment:
seems unrelated to the change -- did you just notice that this was missing?

Reply:
Ah i yeah i just checked the appsignals components.. should have checked all.. 
Maybe later we keep all these in a common reusable place for any aws component..